icu4c/source/data/unit/kl.txt

```diff
@@ -11,8 +11,8 @@ kl{
         length{
             fathom{
                 dnam{"fm"}
-                one{"{0} ftm"}
-                other{"{0} ftm"}
+                one{"{0} fth"}
```

Comment:
Here and a couple other places: why no change to the dnam?

Reply:
Not sure, I will file a separate CLDR ticket about that, thanks. But not a blocker for this merge or for ICU 63 generally.